lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.ts

```diff
@@ -105,7 +105,7 @@ export class SearchChipAutocompleteInputComponent implements OnInit, OnDestroy,
 
     ngOnChanges(changes: SimpleChanges) {
         if (changes.autocompleteOptions) {
-            this.filteredOptions = changes.autocompleteOptions.currentValue.length > 0 ? this.filter(changes.autocompleteOptions.currentValue, this.formCtrl.value) : [];
+            this.filteredOptions = changes.autocompleteOptions.currentValue?.length > 0 ? this.filter(changes.autocompleteOptions.currentValue, this.formCtrl.value) : [];
```

Comment:
Just from curiosity. Did you add '?' because you saw following error in console? 
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/731f07e0-30ad-4616-bc34-299eb0121d48)

If yes then I had similar error and I had discussion with @nikita-web-ua and it was probably because of some missing configuration from Mykyta's jira. If that was a reason please check with @nikita-web-ua if this really needs to be fixed.

Reply:
Yeah I think this was the reason for the change, @nikita-web-ua do I need to add anything additional for this to work?

Reply:
@MichalKinas, @AleksanderSklorz I checked with latest ACA and I do not see an error, have you checked with latest ACA?

Reply:
Will check

Reply:
Okay it works thanks!

Reply:
@AleksanderSklorz @nikita-web-ua guys actually without this null check my unit tests are failing since I'm populating autocompleteOptions in `ngOnChanges` and they're undefined before, would it be okay for you to leave it there?

Reply:
it's small thing so should not be a problem for me